pkgs/development/python-modules/bezier/default.nix

```
@@ -0,0 +1,57 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  buildPythonPackage,
+  setuptools,
+  numpy,
+
+  cmake,
+  gfortran,
+
+  nix-update-script,
+}:
+buildPythonPackage rec {
+  name = "bezier";
+  version = "2024.6.20";
+  src = fetchFromGitHub {
+    owner = "dhermes";
+    repo = "bezier";
+    tag = version;
+    hash = "sha256-TH3x6K5S3uV/K/5e+TXCSiJsyJE0tZ+8ZLc+i/x/fV8=";
+  };
```

```suggestion
  };
  pyproject = true;
```
I don't think that's the default (yet), and it [is recommended to be set to true](https://nixos.org/manual/nixpkgs/unstable/#buildpythonpackage-parameters) if possible.